### PR TITLE
✨ add []aws.ec2.networkinterface to aws.ec2.instance

### PR DESCRIPTION
**Pull request by vjeffrey**
Screenshot 2024-05-27 at 23 05 10

### CODE REVIEW

File: providers/aws/resources/aws.lr.manifest.yaml [hunk 1]
```diff
@@ -1024,6 +1024,8 @@ resources:
       instanceType: {}
       keypair: {}
       launchTime: {}
+      networkInterfaces:
+        min_mondoo_version: latest
```
Comment (tas50) [inline providers/aws/resources/aws.lr.manifest.yaml]:
```suggestion
        min_mondoo_version: 9.0.0
```

File: providers/aws/resources/aws.lr.manifest.yaml [hunk 2]
```diff
@@ -1160,6 +1162,26 @@ resources:
...
+      privateDnsName: {}
+      requesterManaged: {}
+      securityGroups: {}
+      sourceDestCheck: {}
+      status: {}
+      subnet: {}
+      tags: {}
+      vpc: {}
+    is_private: true
+    min_mondoo_version: latest
```
Comment (tas50) [inline providers/aws/resources/aws.lr.manifest.yaml]:
```suggestion
    min_mondoo_version: 9.0.0
```

**Comment by tas50** [comment]
LGTM from the product side

**Review by misterpantz — COMMENTED** [review]
Nice docs! No suggestions. :)

**Review by imilchev — APPROVED** [review]
everyone seems to agree this is good to go!

**Pull request merged**